apps/andi/lib/andi/input_schemas/input_converter.ex

```diff
@@ -483,7 +483,6 @@ defmodule Andi.InputSchemas.InputConverter do
     |> Map.delete(:id)
     |> Map.delete(:dataset_id)
     |> Map.delete(:bread_crumb)
-    |> Map.delete(:ingestion_field_sync)
```

Comment:
Is this the main reason for the ingestion_field_sync state not being persisted on refresh?

Reply:
this was, I thought this was going to other places besides just Ecto, so was my b last pass